src/dyp_a01.toit

```diff
@@ -30,27 +31,22 @@ class DYP_A01:
     port.close 
 
   range -> int:                 // Range to target, in mm
-      val := range_
-      if val < 0: val = range_  // retry 3 times, to resync frames
-      if val < 0: val = range_
-      if val < 0: val = range_
-      return val
+    val := range_
+    if val < 0: val = range_  // retry 3 times, to resync frames
```

Comment:
Didn't change it, but we generally prefer comments to start capitalized and finish with a dot:
```suggestion
    if val < 0: val = range_  // Retry 3 times, to resync frames.
```
It just wasn't important enough to change.